tutoraspects/plugin.py

```diff
@@ -255,6 +255,12 @@
                 "student": "Student",
             },
         ),
+        (
+            "SUPERSET_EMBEDDABLE_DASHBOARDS",
+            {
+                "instructor-dashboard": "1d6bf904-f53f-47fd-b1c9-6cd7e284d286",
```

Comment:
Does this default make sense to you? Or should I leave it empty with a comment on how to use this setting?

Reply:
I assume that marking it as embeddable won't change it's permissions, you'd still need to be logged in as someone of the correct group to see it, and it would still filter results based on your RLSF?

Reply:
External users will need to API credentials to view the dashboard in other sites, so we are safe.

And yes, permissions are the same

Reply:
👍